src/digest/digest.rs

```diff
@@ -292,24 +296,6 @@ pub struct Algorithm {
     pub nid: c::int,
 }
 
-/// SHA-1 as specified in [FIPS
```

Comment:
I worry that if we move this to the submodule, any time we need to change all of the `Algorithm` specifications, we risk forgetting to change SHA-1's. So, I'd rather leave this here.